lowlevel/src/motor-board/main.cpp

```diff
@@ -66,79 +66,53 @@ int main() {
 
 
     // Order reception
-    if (canpb.is_rx_available()) {
-      rx_packet++;
-      if (canpb.receive_msg(msg_rx)) {
-        switch (msg_rx.which_message_content) {
-          case BusMessage_stopMoving_tag:serial.println("Packet Received: StopMoving");
-            mcs.stop(false);
-            break;
-
-          case BusMessage_translate_tag:serial.println("Packet Received: Translate");
-            mcs.translate_ticks(msg_rx.message_content.translate.ticks);
-            break;
-
-          case BusMessage_rotate_tag:serial.println("Packet Received: Rotate");
-            mcs.rotate_ticks(msg_rx.message_content.rotate.ticks);
-            break;
-
-          case BusMessage_moveWheelAtSpeed_tag:serial.println("Packet Received: MoveWheelAtSpeed");
-            mcs.set_target_speed(Motor::Side::LEFT,
-                                 msg_rx.message_content.moveWheelAtSpeed.left_tick_per_sec);
-            mcs.set_target_speed(Motor::Side::RIGHT,
-                                 msg_rx.message_content.moveWheelAtSpeed.right_tick_per_sec);
-            break;
-
-          case BusMessage_setMotionControlMode_tag:serial.println("Packet Received: SetMotionControlMode");
-            mcs.set_control(msg_rx.message_content.setMotionControlMode.speed,
-                            msg_rx.message_content.setMotionControlMode.translation,
-                            msg_rx.message_content.setMotionControlMode.rotation);
-            break;
-
-          case BusMessage_pidConfig_tag:serial.println("Packet Received: PID Config");
-            mcs.set_kp(msg_rx.message_content.pidConfig.pid_id, msg_rx.message_content.pidConfig.kp);
-            mcs.set_ki(msg_rx.message_content.pidConfig.pid_id, msg_rx.message_content.pidConfig.ki);
-            mcs.set_kd(msg_rx.message_content.pidConfig.pid_id, msg_rx.message_content.pidConfig.kd);
-            break;
-
-          case BusMessage_motionLimit_tag:serial.println("Packet Received: Motion Limits");
-            mcs.set_limits(
-                msg_rx.message_content.motionLimit.translation_speed,
-                msg_rx.message_content.motionLimit.rotation_speed,
-                msg_rx.message_content.motionLimit.wheel_speed,
-                msg_rx.message_content.motionLimit.wheel_acceleration
-            );
-            break;
-
-          default:break;
-        }
+    if (canpb.receive_msg(msg_rx)) {
+      nb_packets_rx++;
+      switch (msg_rx.which_message_content) {
+        case BusMessage_moveWheelAtSpeed_tag:
+          mcs.set_speed_targets(msg_rx.message_content.moveWheelAtSpeed.left_tick_per_sec,
+                                msg_rx.message_content.moveWheelAtSpeed.right_tick_per_sec);
+          keepalive_timer.reset();
```

Comment:
you can move this out of the switch case and reset it for every BusMessage received

Reply:
Done in 8034602